### PR TITLE
CLDR-13431 v36.1 final: update readme; copy emoji collation & annotations from v37

### PR DESCRIPTION
**Pull request by pedberg-icu**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13431
- [x] Updated PR title and link in previous line to include Issue number

Final changes for CLDR v36.1 (on maint/maint-36 branch):
1. Update readme.html date and to say final release
2. Copy emoji collation and annotations (and some related items) from CLDR v37, specifically from master @ d3ffc1b437c7b51ca5dad0051ac7dbd5674aa785 (though each "git checkout master <path>" for these files reported "Updated 1 path from a208afe7e"):
  - Copy common/collation/root.xml; the only thing that changed was the emoji collation. This is the only thing that will affect ICU.
  - Copy the updated contents of common/annotations/ and common/annotationsDerived/ to get updated annotations in updated order. However: the updated annotations in common/annotations/ include not  only updated emoji annotations but also new annotations for symbols which are a CLDR v37 thing not related to Unicode 13. So deleted these; they also caused test failures with the CLDR v36.1 tests. Fortunately these were in two contiguous blocks in the annotations data, the first from &lt;annotation cp="–"&gt; to &lt;annotation cp="◯"&gt;, and the second from &lt;annotation cp="£"&gt; to &lt;annotation cp="µ"&gt;.
  - In order for annotation tests to pass, also need to have the new emoji images; so also copy the new and updated images from tools/cldr-apps/WebContent/images/emoji/
  - In order for the annotation tests to pass, also need to have the updated emoji-test.txt file from tools/java/org/unicode/cldr/util/data/emoji/
  - In order for the annotation tests to pass, also need to update the SPECIALS set in tools/java/org/unicode/cldr/util/Emoji.java, so just copy that set from the v37 version
  - In order for the emoji ordering tests to pass, need to update the desiredOrder[] array in tools/cldr-unittest/src/org/unicode/cldr/unittest/TestPathHeader.java, so just copy that array from the v37 version

### CODE REVIEW

File: common/collation/root.xml [hunk 1]
```diff
@@ -947,20 +947,20 @@ For terms of use, see http://www.unicode.org/copyright.html
 &龠=\uFDD0龠 # radical 214
 				]]></cr>
 			</collation>
-<!-- Machine-readable version of the emoji ordering rules for v12.0 (corresponding to CLDR). -->
+<!-- Machine-readable version of the emoji ordering rules for v13.0 (corresponding to CLDR). -->
 <collation type='emoji'>
 <cr><![CDATA[
 # START AUTOGENERATED EMOJI ORDER
 & [last primary ignorable]<<*🦰🦱🦳🦲🏻🏼🏽🏾🏿
 & [before 1]﷑€
```
Comment (markusicu) [inline common/collation/root.xml]: Is that a U+FFFD before the € sign? Does that work with the ICU Collator?

Comment (pedberg-icu) [reply common/collation/root.xml]: It is U+FDD1 and is already present in the ICU maint/maint-66 collation data, so no change there.

Comment (markusicu) [reply common/collation/root.xml]: Oh, that makes sense.
Sorry, I just looked at it with a browser extension for Unicode properties, and I guess that turns noncharacter code points into U+FFFD for some reason...
Should have looked at the actual file.

Comment (markusicu) [reply common/collation/root.xml]: PS: It would be clearer to have that one be escaped in the collation rules, as \uFDD1

Comment (macchiati) [reply common/collation/root.xml]: Good idea.

(I want to get unicodetools moved to github after the release; then we can keep track of improvements like this!)

**Pull request merged**